install/helm/gloo/generate/values.go

```diff
@@ -437,7 +437,8 @@ type K8s struct {
 }
 
 type Stats struct {
-	Enabled bool `json:"enabled,omitempty" desc:"Controls whether or not prometheus stats are enabled"`
+	Enabled            bool    `json:"enabled,omitempty" desc:"Controls whether or not envoy stats are enabled"`
+	RoutePrefixRewrite *string `json:"routePrefixRewrite,omitempty" desc:"The envoy stats endpoint to which the metrics endpoint is rewritten"`
```

Comment:
nit, grammar
```suggestion
	RoutePrefixRewrite *string `json:"routePrefixRewrite,omitempty" desc:"The envoy stats endpoint to which the metrics are written"`
```
this will require codegen again, since this ends up in our generated helm docs